### ISSUE TITLE
explanation of the df from subsetCommunication

### ISSUE THREAD

**Issue by amberqiong** [sqjin/CellChat#750]
Hello,

Thanks for the wonderful cellchat tool! I have some questions regarding regarding the explanation of the dataframe from `subsetCommunication()`. Does the 'prob' column represent the communication probability (strength)? Can I directly compare this value between two datasets to assess if the signaling is up-regulated or down-regulated?

`lipo.control.signaling.df=subsetCommunication(object.list[[1]],sources.use = c(1:3),targets.use = c(1:3))
lipo.PA.signaling.df=subsetCommunication(object.list[[2]],sources.use = c(1:3),targets.use = c(1:3))`

My goal is to identify the list of the differential signaling, and then comparing to other dataset (find common changed pathways).

Thanks in advance.

**Comment by sqjin** [comment]
@amberqiong Yes, you can compare the value between two datasets. the 'prob' column represent the communication probability (strength)

**Comment by amberqiong** [comment]
Thank you for the clarification!

> @amberqiong Yes, you can compare the value between two datasets. the 'prob' column represent the communication probability (strength)

**Comment by amberqiong** [comment]
Is there a defined threshold for the 'prob' column that signifies strong communication?

**Comment by sqjin** [comment]
@amberqiong no threshold is for `prob`.  But high values indicate strong communicaitons
